plugins/Wallet/js/components/transactionlist.js

```diff
@@ -25,8 +25,8 @@ const TransactionList = ({transactions}) => {
 		}
 		return (
 			<tr key={key}>
-				<td>{valueData}</td>
-				<td>{txn.transactionid}</td>
+				<td className="txval">{valueData}</td>
```

Comment:
ah, I originall was formatting the value as monospace as well. I think it only makes sense to monospace the IDs, so I've removed the `txval` class.